data/korath/korath missions.txt

```diff
@@ -257,3 +257,606 @@ mission "Small Scale Delivery"
+				decline
+
+			label alone
+			`	On the map, you draw a line between Korath space and human space. "Humans will stay here," you say, "if the Korath will stay here. Humans here, Korath here." You point to each location for emphasis.`
+				goto parting
+			label fight
+			`	It pushes its finger deep into the ground at a point on the map not too far from Earth. "Ka puk soru, nik ka korath."`
+			label parting
+			`	"Kai," says the Korath, and it stretches out both arms toward you, palms out. You make the same gesture and touch its hands, palm to palm. You're not sure if that is the right response, but the Korath seem satisfied, saying, "Nik lak'mal." They get back in their fighters, take off, and hover nearby.`
+				decline
+
+
+
+mission "Korath [F01]: A Fishy Deal"
+	landing
+	name "A fishy proposition"
+	description "Transport <cargo> to <planet> by <date>. After a return trip, you'll be paid 140,000. Don't get caught."
+	cargo "live fish" 15
+	# The mission is illegal because of the broader trade agreement. The player is supposed to be left with the impression that the Korath are sending back more dangerous things than waste processing systems. The Syndicate (and by extension, the Republic) is aware of this, and fines pilots involved.
+	illegal 50000
+	deadline 4
+	clearance "The authorities do not respond, but a concealed hangar bay door opens. The meaning is clear: land here."
+	source "New Tortuga"
+	destination "La'ab Bamen"
+	on offer
+		event "know about Korath fish jobs"
+		conversation
+			`Flying through the atmosphere of New Tortuga, you pass over a typhoon, overfly a tsunami, and take a short detour around a supercell thunderstorm. After a half hour of delicate flying, you approach doors more than a hundred meters tall. As they open, you can see they're ten meters thick, made of a material you don't recall seeing on human planets. Two layers of shields shut down in anticipation of your entry, each strong enough to stop a Navy Carrier. To the left and right, there are turrets with heat signatures higher than a Breeder Reactor. Whoever built this place values their safety... and has the funds and technology to secure it.`
+			`	Spaceport authorities direct you to bay 23X-5, which contains an odd ship you see on your way in. It appears to be an old-fashioned jet aircraft, whose four jet engines have been replaced with small ion engines. From the gold pipes on the tail, it appears to have a hyperdrive built into the hull. Several packages are being unloaded from the plane. A woman is waving her arms, perhaps with anger, as she walks from the cockpit into the airport.`
+			`	The ship is parked next to yours and the woman is about to walk past you. She looks quite distraught, and perchance looks your way. Maybe a distraction will cheer her up?`
```

Comment:
```suggestion
			`	The ship is parked next to yours and the woman is about to walk past you. She looks quite distraught and, perchance, looks your way. Maybe a distraction will cheer her up?`
```

This could help emphasize that intent of ambiguity here.